docker-compose.yml

```
@@ -166,7 +166,7 @@ services:
       - transformation_grids:/transformation_grids
       - /var/run/docker.sock:/var/run/docker.sock
       - ${LOG_DIRECTORY}:/log
-      - ${TMP_DIRECTORY}:/tmp
+      - ${QFIELDCLOUD_QGIS_TMP_DIRECTORY:-/tmp}:/tmp
```

Added default as `/tmp` here, if the env var is not present

---

I think it's a bit too dangerous to add a default here, would you mind removing it?


```suggestion
      - ${QFIELDCLOUD_QGIS_TMP_DIRECTORY}:/tmp
```